R/pkg/R/deserialize.R

```diff
@@ -235,15 +235,18 @@ readDeserializeInArrow <- function(inputCon) {
   # This is a hack to avoid CRAN check. Arrow is not uploaded into CRAN now. See ARROW-3204.
   requireNamespace1 <- requireNamespace
   requireNamespace1("arrow", quietly = TRUE)
+  RecordBatchStreamReader <- get(
+    "RecordBatchStreamReader", envir = asNamespace("arrow"), inherits = FALSE)
+  as_tibble <- get("as_tibble", envir = asNamespace("arrow"))
```

Comment:
these are pretty nasty... maybe put all these into a R file/method

```
  requireNamespace1 <- requireNamespace
  requireNamespace1("arrow", quietly = TRUE)
  RecordBatchStreamReader <- get(
    "RecordBatchStreamReader", envir = asNamespace("arrow"), inherits = FALSE)
  as_tibble <- get("as_tibble", envir = asNamespace("arrow"))
```

Reply:
Yes .. I actually tried to pull those out.. but I ended up with leave them as are .. I target to remove those weird hacks as soon as Arrow is on CRAN which I estimate before Spark 3.0. I promise I clean up those before Spark 3.0.